File: src/test/java/test/GetExtIdpTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetExtIdpTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetExtIdpDto request = new GetExtIdpDto();
        request.setId("id_2120");
        request.setTenantId("tenantId_3495");

        ExtIdpDetailSingleRespDto response = client.getExtIdp(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetExtIdpTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetExtIdpDto{
        Id: "id_4241",
        TenantId: "tenantId_9898",
        AppId: "appId_1548",
        Type: "type_8357",
    }
    response := client.GetExtIdp(&request)
    fmt.Println(response)
}

